code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -849,3 +853,10 @@
 					else
 						to_chat(user, "<span class='cultitalic'>You need a free hand for this rite!</span>")
 						qdel(rite)
+
+/obj/item/melee/blood_magic/manipulator/proc/check_menu(mob/living/user)
+	if(!istype(user))
+		return FALSE
```

Comment:
```suggestion
```
Avoid sanity checks.
If you have a reason to think this might happen when it shouldn't, add a `CRASH()` or `stack_trace()` message instead of failing silently.